rel/config/console.exs

```diff
@@ -22,7 +22,7 @@ config :libcluster,
       config: [
         mode: :ip,
         kubernetes_node_basename: "console",
-        kubernetes_selector: "app.kubernetes.io/name=console",
```

Comment:
I think this is actually the preferred notation over `app=console`. Could also make this change in the helm chart if needed otherwise.

Reply:
they're basically the same, I think.  The service has both labels set